examples/eSealing/src/lib.rs

```diff
@@ -0,0 +1,337 @@
+//! A contract implementing an eSealing service.
+//!
+//! If you seal a file, you can prove that it was in your possession at the time
+//! of sealing. The dApp flow will be:
+//! - Upload a file from the computer at the front-end => register its file hash
+//!   in the smart contract
+//! - Upload a file from the computer at the front-end => retrieve the timestamp
+//!   and witness (sender_account) from the smart contract to prove that the
+//!   witness (sender_account) was in possession of that file at that time.
+//!
+//! Only accounts can register a file hash. During the registration, the
+//! timestamp is recorded together with the witness (sender_account) that
+//! registered the file hash. Each file hash can only be registered once.
+//! This is because we want to record the first time when the witness
+//! (sender_account) had access to that file. Re-registering a file hash (by a
+//! different witness) would not prove that the second witness is also in
+//! possession of that file or that it just observed the file hash during the
+//! initial registration hence a file hash can only be registered once.
```

Comment:
The part "or that it just observed" I don't understand. Is there something off in the sentence?